ndcube/global_coords/global_coords.py

```diff
@@ -28,6 +30,20 @@ def _all_coords(self):
         A dynamic dictionary of all global coordinates, stored here or derived
         from the ndcube object.
         """
+        if self._ndcube is None:
+            return self._internal_coords
+
+        if hasattr(self._ndcube.wcs.low_level_wcs, "dropped_world_dimensions"):
+            dropped_world = self._ndcube.wcs.low_level_wcs.dropped_world_dimensions
+            extra_internal = {}
```

Comment:
Changing this name to be a little more descriptive would be good, maybe:
```suggestion
            wcs_dropped = {}
```